src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java

```diff
@@ -132,26 +141,28 @@ public void setIgnoreEnums(boolean ignoreEnums) {
     @Override
     public int[] getAcceptableTokens() {
         return new int[] {
-            TokenTypes.INTERFACE_DEF,
-            TokenTypes.CLASS_DEF,
```

Comment:
please make sorting in separate/first commit of this PR (two commits in much better).
We try to keep formatting aside of functional change.

Reply:
done